src/frameworks/remix.json

```diff
@@ -0,0 +1,20 @@
+{
+  "id": "remix",
+  "name": "Remix",
+  "category": "static_site_generator",
+  "detect": {
+    "npmDependencies": ["remix", "@remix-run/netlify"],
+    "excludedNpmDependencies": [],
+    "configFiles": ["remix.config.mjs"]
+  },
+  "dev": {
+    "command": "remix watch"
+  },
+  "build": {
+    "command": "remix build",
+    "directory": "public"
+  },
+  "staticAssetsDirectory": "public",
```

Comment:
I'm not completely sure what `staticAssetsDirectory` does, and how it differs from `build.directory`

Reply:
It's explained in https://github.com/netlify/framework-info/issues/103.

For Remix looks like we should remove it

Reply:
Done in https://github.com/netlify/framework-info/pull/658/commits/9ae44ec50996b85744538cde89e836a96bab99bc